tests/Unit/CacheDirectoryFactory/CacheDirectoryFactoryTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Crunz\Tests\Unit\CacheDirectoryFactory;
+
+use Crunz\CacheDirectoryFactory\CacheDirectoryFactory;
+use Crunz\Path\Path;
+use PHPUnit\Framework\TestCase;
+
+class CacheDirectoryFactoryTest extends TestCase
```

Comment:
Could You use `AAA` [approach](https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/) in both test classes?